Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -40,6 +41,7 @@ async function run() {
         var swapWithProduction = tl.getBoolInput('SwapWithProduction', false);
         var targetSlot: string = tl.getInput('TargetSlot', false);
         var preserveVnet: boolean = tl.getBoolInput('PreserveVnet', false);
+        var extensionList = tl.getInput('ExtensionsList', false);
```

Comment:
To confirm - This input will have the list of extension Id is it? If so, if you plan to support dynamic pick list for this input, will it still remain as Id?